<html><body>
<h1>ToppGene analysis for HTSeq_genes_nodedupnosubs_noR_noDupl_Log</h1><br><br><br>
<table width=100% border=1>
<tr><td width=10%>POSITIVE SIDE</td>
<td width=30%>ANNOTATION</td>
<td width=1%>*<br>*</td>
<td width=10%>NEGATIVE SIDE</td>
<td width=30%>ANNOTATION</td></tr>
<tr>
<td width=10%><a href="IC1plus.html">IC1+ </a>(<a href="IC1plus_genes.html">18 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/RUNNE_GENDER_EFFECT_UP.html">Up-regulated genes detecting gender effects in global expression profiling studies.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">ratio EmbryoidBody-fibro vs EmbryoidBody-derived-from CD34-iPSC top-relative-expression-ranked 100</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC1minus.html">IC1- </a>(<a href="IC1minus_genes.html">12 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC2plus.html">IC2+ </a>(<a href="IC2plus_genes.html">106 genes</a>)</td>
<td width=30%>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS50835">IG LIKE</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC2minus.html">IC2- </a>(<a href="IC2minus_genes.html">107 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC3plus.html">IC3+ </a>(<a href="IC3plus_genes.html">195 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0017171">serine hydrolase activity</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0008202">steroid metabolic process</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0072562">blood microparticle</a><br>4: Human Phenotype:<a href="null">Abnormality of the common coagulation pathway</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0002551">abnormal blood coagulation</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR017972">Cyt P450 CS</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_COMPLEMENT_AND_COAGULATION_CASCADES.html">Complement and coagulation cascades</a><br>9: Interaction:<a href=" ">GDPD1 interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$HNF1_Q6">V$HNF1 Q6</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HSIAO_LIVER_SPECIFIC_GENES.html">Liver selective genes</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:liver;+AND+sort_by:tissue+specific+score">liver</a><br>18: Disease:<a href="">Adverse reaction to drug</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC3minus.html">IC3- </a>(<a href="IC3minus_genes.html">42 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC4plus.html">IC4+ </a>(<a href="IC4plus_genes.html">374 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0003779">actin binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0002576">platelet degranulation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0031091">platelet alpha granule</a><br>4: Human Phenotype:<a href="null">Increased mean platelet volume</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0002551">abnormal blood coagulation</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1269367">Platelet degranulation</a><br>9: Interaction:<a href=" ">MYH9 interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$SRF_01">V$SRF 01</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/17192395">Human StemCell Macaulay07 516genes</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP MEG top-relative-expression-ranked 500 k-means-cluster#4</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-7087-3p"></a><br>18: Disease:<a href="">Macrothrombocytopenia</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%>IC4- (<a href="IC4minus_genes.html">3 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC5plus.html">IC5+ </a>(<a href="IC5plus_genes.html">125 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006959">humoral immune response</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/KIM_GLIS2_TARGETS_UP.html">Partial list of genes up-regulated in the kidney of GLIS2 [GeneID=84662] knockout mice compared to the wild type.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777046">Stromal Cells, Fi.Sk, gp38+ CD140a+, Skin, avg-4</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC5minus.html">IC5- </a>(<a href="IC5minus_genes.html">96 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/20371479">Human Leukemia Saito10 259genes</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 2500 k-means-cluster#2</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC6plus.html">IC6+ </a>(<a href="IC6plus_genes.html">138 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0001228">DNA-binding transcription activator activity, RNA polymerase II-specific</a><br>18: Disease:<a href="">Urinary tract infection</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC6minus.html">IC6- </a>(<a href="IC6minus_genes.html">124 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/TAKEDA_TARGETS_OF_NUP98_HOXA9_FUSION_10D_UP.html">Genes up-regulated in CD34+ [GeneID=947] hematopoetic cells by expression of NUP98-HOXA9 fusion [GeneID=4928;3205] off a retroviral vector at 10 days after transduction.</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC7plus.html">IC7+ </a>(<a href="IC7plus_genes.html">93 genes</a>)</td>
<td width=30%>none</td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC7minus.html">IC7- </a>(<a href="IC7minus_genes.html">96 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC8plus.html">IC8+ </a>(<a href="IC8plus_genes.html">149 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0050954">sensory perception of mechanical stimulus</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0001363">increased anxiety-related response</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/KIM_GLIS2_TARGETS_UP.html">Partial list of genes up-regulated in the kidney of GLIS2 [GeneID=84662] knockout mice compared to the wild type.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium EmbryoidBody fromBlastocyst-derived-humanEmbryonicStemCells top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-212-5p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC8minus.html">IC8- </a>(<a href="IC8minus_genes.html">126 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.facebase.org">FacebaseRNAseq e10.5 Olfactory Pit top-relative-expression-ranked 500 k-means-cluster#3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-6971-5p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC9plus.html">IC9+ </a>(<a href="IC9plus_genes.html">343 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0022839">ion gated channel activity</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030198">extracellular matrix organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005903">brush border</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0002136">abnormal kidney physiology</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR013032">EGF-like CS</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE29618_BCELL_VS_PDC_DAY7_FLU_VACCINE_DN.html">Genes down-regulated in comparison of B cells from influenza vaccinee at day 7 post-vaccination versus plasmacytoid dendritic cells (pDC) at day 7 post-vaccination.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM605840">Plasmacytoid, DC.pDC.8+.Sp, GR1-PE CD11c-eFluor750 CD8-eFluor480 CD45-APC B220, Spleen, avg-3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4640-5p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%>IC9- (<a href="IC9minus_genes.html">7 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%>---</td>
<td width=30%>---</td>
<td width=1%>*<br>*</td>
<td width=10%>---</td>
<td width=30%>---</td>
</tr>
<tr>
<td width=10%><a href="IC11plus.html">IC11+ </a>(<a href="IC11plus_genes.html">145 genes</a>)</td>
<td width=30%>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/GATAAGR_V$GATA_C">GATAAGR V$GATA C</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HALLMARK_HEME_METABOLISM.html">Genes involved in metabolism of heme (a cofactor consisting of iron and porphyrin) and erythroblast differentiation.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP RBC top-relative-expression-ranked 500 k-means-cluster#2</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4663"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC11minus.html">IC11- </a>(<a href="IC11minus_genes.html">128 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030198">extracellular matrix organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MIKKELSEN_MCV6_HCP_WITH_H3K27ME3.html">Genes with high-CpG-density promoters (HCP) bearing the tri-methylation mark at H3K27 (H3K27me3) in MCV6 cells (embryonic fibroblasts trapped in a differentiated state).</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">JC fibro top-relative-expression-ranked 500</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC12plus.html">IC12+ </a>(<a href="IC12plus_genes.html">315 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0050953">sensory perception of light stimulus</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:1902495">transmembrane transporter complex</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0008948">decreased neuron number</a><br>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS00022">EGF 1</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1383050">Defective C1GALT1C1 causes Tn polyagglutination syndrome (TNPS)</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/GGATTA_V$PITX2_Q2">GGATTA V$PITX2 Q2</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/648">Mucins</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MIKKELSEN_MCV6_HCP_WITH_H3K27ME3.html">Genes with high-CpG-density promoters (HCP) bearing the tri-methylation mark at H3K27 (H3K27me3) in MCV6 cells (embryonic fibroblasts trapped in a differentiated state).</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium induced-Ectoderm fromBlastocyst-derived-humanEmbryonicStemCells top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4640-5p"></a><br>18: Disease:<a href="">Disorder of eye</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC12minus.html">IC12- </a>(<a href="IC12minus_genes.html">18 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC13plus.html">IC13+ </a>(<a href="IC13plus_genes.html">258 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019955">cytokine binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0043312">neutrophil degranulation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0042581">specific granule</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0008568">abnormal interleukin secretion</a><br>6: Domain:<a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=IG">IG</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1457780">Neutrophil degranulation</a><br>9: Interaction:<a href=" ">JUNB interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$AML_Q6">V$AML Q6</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HALLMARK_TNFA_SIGNALING_VIA_NFKB.html">Genes regulated by NF-kB in response to TNF [GeneID=7124].</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM854338">Myeloid Cells, GN.Arth.SynF, CD11b+ Ly6-G+, Synovial Fluid, avg-3</a><br>16: MicroRNA:<a href=" ">Functional MTI</a><br>18: Disease:<a href="">Juvenile psoriatic arthritis</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC13minus.html">IC13- </a>(<a href="IC13minus_genes.html">30 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC14plus.html">IC14+ </a>(<a href="IC14plus_genes.html">233 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0031720">haptoglobin binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030218">erythrocyte differentiation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005833">hemoglobin complex</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0000226">abnormal mean corpuscular volume</a><br>10: Cytoband:<a href=" ">19p13.3</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/GATAAGR_V$GATA_C">GATAAGR V$GATA C</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/454">Blood group antigens</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HALLMARK_HEME_METABOLISM.html">Genes involved in metabolism of heme (a cofactor consisting of iron and porphyrin) and erythroblast differentiation.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP RBC top-relative-expression-ranked 1000 k-means-cluster#4</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-5627-5p"></a><br>18: Disease:<a href="">beta^+^ Thalassemia</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC14minus.html">IC14- </a>(<a href="IC14minus_genes.html">34 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC15plus.html">IC15+ </a>(<a href="IC15plus_genes.html">165 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0003823">antigen binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006958">complement activation, classical pathway</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019814">immunoglobulin complex</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR003006">Ig/MHC CS</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1269280">FCGR activation</a><br>9: Interaction:<a href=" ">SNX27 interactions</a><br>10: Cytoband:<a href=" ">14q32.33</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/349">Immunoglobulin heavy locus at 14q32.33</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MCLACHLAN_DENTAL_CARIES_UP.html">Genes up-regulated in pulpal tissue extracted from carious teeth.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">AravindRamakr StemCell-HD top-relative-expression-ranked 1000 k-means-cluster#3</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC15minus.html">IC15- </a>(<a href="IC15minus_genes.html">66 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium inducedDefinitiveEndoderm fromFibroblast-derived-iPSC top-relative-expression-ranked 500</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC16plus.html">IC16+ </a>(<a href="IC16plus_genes.html">247 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0090130">tissue migration</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR009030">Growth fac rcpt</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$SOX9_B1">V$SOX9 B1</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/593">Fibronectin type III domain containing|I-set domain containing|Immunoglobulin like domain containing</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/JAATINEN_HEMATOPOIETIC_STEM_CELL_UP.html">Genes up-regulated in CD133+ [GeneID=8842] cells (hematopoietic stem cells, HSC) compared to the CD133- cells.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777032">Stromal Cells, BEC.MLN, gp38- CD31+, Lymph Node, avg-5</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4314"></a><br>18: Disease:<a href="">Desmoplastic</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC16minus.html">IC16- </a>(<a href="IC16minus_genes.html">79 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.facebase.org">FacebaseRNAseq e10.5 Neural Epithelium Overlying Lateral Eminence top-relative-expression-ranked 2500 k-means-cluster#3</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC17plus.html">IC17+ </a>(<a href="IC17plus_genes.html">144 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://code.google.com/p/altanalyze/wiki/PCBC_C4_compendium">Embryoid Body Cells-reprogram OSKM-L vs Embryoid Body Cells-reprogram NA-Confounder removed-fold2.0 adjp0.05</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-5682"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC17minus.html">IC17- </a>(<a href="IC17minus_genes.html">145 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:1901617">organic hydroxy compound biosynthetic process</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR013098">Ig I-set</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/593">Fibronectin type III domain containing|I-set domain containing|Immunoglobulin like domain containing</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium reference SmallAirwayEpithel top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-6742-3p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC18plus.html">IC18+ </a>(<a href="IC18plus_genes.html">152 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HSIAO_LIVER_SPECIFIC_GENES.html">Liver selective genes</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:liver;+AND+sort_by:tissue+specific+score">liver</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-637"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC18minus.html">IC18- </a>(<a href="IC18minus_genes.html">144 genes</a>)</td>
<td width=30%>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0045111">intermediate filament cytoskeleton</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MCBRYAN_PUBERTAL_BREAST_3_4WK_UP.html">Genes up-regulated during pubertal mammary gland development between weeks 3 and 4.</a><br>14: Coexpression Atlas:<a href=" ">BM Top 100 - trachea</a><br>16: MicroRNA:<a href="http://www.targetscan.org/">hsa-miR-212-5p</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC19plus.html">IC19+ </a>(<a href="IC19plus_genes.html">224 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0034987">immunoglobulin receptor binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0050853">B cell receptor signaling pathway</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019814">immunoglobulin complex</a><br>4: Human Phenotype:<a href="null">Immunoglobulin abnormality</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0002144">abnormal B cell differentiation</a><br>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS50835">IG LIKE</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1269184">Antigen activates B Cell Receptor (BCR) leading to generation of second messengers</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$ETS_Q4">V$ETS Q4</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE29618_BCELL_VS_MONOCYTE_UP.html">Genes up-regulated in comparison of B cells versus monocytes.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM538219">B cells, B.T3.Sp, IgD+ IgM+ CD45R+ CD24+ CD19+ AA4.1+ CD23+, Spleen, avg-3</a><br>16: MicroRNA:<a href="http://www.targetscan.org/">hsa-miR-874-3p</a><br>18: Disease:<a href="">Waldenstrom Macroglobulinemia</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC19minus.html">IC19- </a>(<a href="IC19minus_genes.html">86 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/JAATINEN_HEMATOPOIETIC_STEM_CELL_UP.html">Genes up-regulated in CD133+ [GeneID=8842] cells (hematopoietic stem cells, HSC) compared to the CD133- cells.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP epoMEP top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4308"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC20plus.html">IC20+ </a>(<a href="IC20plus_genes.html">212 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0003735">structural constituent of ribosome</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0000184">nuclear-transcribed mRNA catabolic process, nonsense-mediated decay</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0022626">cytosolic ribosome</a><br>4: Human Phenotype:<a href="null">Congenital hypoplastic anemia</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1268688">L13a-mediated translational silencing of Ceruloplasmin expression</a><br>9: Interaction:<a href=" ">RPS4X interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$NRF2_01">V$NRF2 01</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/729">L ribosomal proteins</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HSIAO_HOUSEKEEPING_GENES.html">Housekeeping genes identified as expressed across 19 normal tissues.</a><br>14: Coexpression Atlas:<a href="http://www.facebase.org">FacebaseRNAseq ratio e10.5 MandibularArch vs MaxillaryArch top-relative-expression-ranked 1000 k-means-cluster#2</a><br>16: MicroRNA:<a href=" ">Functional MTI (Weak)</a><br>18: Disease:<a href="">Anemia, Diamond-Blackfan</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC20minus.html">IC20- </a>(<a href="IC20minus_genes.html">86 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC21plus.html">IC21+ </a>(<a href="IC21plus_genes.html">169 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0048705">skeletal system morphogenesis</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0002932">abnormal joint morphology</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR017970">Homeobox CS</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1269010">Diseases of glycosylation</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/518">HOXL subclass homeoboxes</a><br>14: Coexpression Atlas:<a href="http://www.gudmap.org/">kidney P4 CapMesRenVes Crym top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="">hsa-miR-1248:PITA TOP</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC21minus.html">IC21- </a>(<a href="IC21minus_genes.html">86 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC22plus.html">IC22+ </a>(<a href="IC22plus_genes.html">300 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0043312">neutrophil degranulation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0042581">specific granule</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0004046">abnormal mitosis</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1457780">Neutrophil degranulation</a><br>9: Interaction:<a href=" ">FZR1 interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$E2F_Q3">V$E2F Q3</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE6269_E_COLI_VS_STREP_PNEUMO_INF_PBMC_DN.html">Genes down-regulated in comparison of peripheral blood mononuclear cells (PBMC) from patients with acute E. coli infection versus PBMC from patients with acute S. pneumoniae infection.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777030">B cells, B.Pl.AA4-.BM, CD138+ AA4.1- CD43+, Bone marrow, avg-2</a><br>16: MicroRNA:<a href=" ">Functional MTI (Weak)</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC22minus.html">IC22- </a>(<a href="IC22minus_genes.html">35 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC23plus.html">IC23+ </a>(<a href="IC23plus_genes.html">149 genes</a>)</td>
<td width=30%>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4656"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC23minus.html">IC23- </a>(<a href="IC23minus_genes.html">125 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC24plus.html">IC24+ </a>(<a href="IC24plus_genes.html">155 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030198">extracellular matrix organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/20599952">Human BoneMarrow Maisel10 105genes</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM854302">Stromal Cells, Ep.MEChi.Th, CD45-EpCAm+Ly51-MHCIIhi, Thymus, avg-1</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-675&organism=9606">hsa-miR-675:mirSVR nonconserved lowEffect-0.1-0.5</a><br>18: Disease:<a href="">Influenza</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC24minus.html">IC24- </a>(<a href="IC24minus_genes.html">139 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MEISSNER_BRAIN_HCP_WITH_H3K27ME3.html">Genes with high-CpG-density promoters (HCP) bearing the H3K27 tri-methylation (H3K27me3) mark in brain.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP RBC top-relative-expression-ranked 1000 k-means-cluster#4</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-1203&organism=9606">hsa-miR-1203:mirSVR nonconserved lowEffect-0.1-0.5</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC25plus.html">IC25+ </a>(<a href="IC25plus_genes.html">137 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006956">complement activation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019814">immunoglobulin complex</a><br>4: Human Phenotype:<a href="null">Avascular necrosis</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1268734">O-linked glycosylation</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-12205-5p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC25minus.html">IC25- </a>(<a href="IC25minus_genes.html">137 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MEISSNER_BRAIN_HCP_WITH_H3K27ME3.html">Genes with high-CpG-density promoters (HCP) bearing the H3K27 tri-methylation (H3K27me3) mark in brain.</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4660"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC26plus.html">IC26+ </a>(<a href="IC26plus_genes.html">130 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0038024">cargo receptor activity</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/SCHUETZ_BREAST_CANCER_DUCTAL_INVASIVE_UP.html">Genes up-regulated in invasive ductal carcinoma (IDC) relative to ductal carcinoma in situ (DCIS, non-invasive).</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777050">Stromal Cells, FRC.MLN, gp38+ CD31- CD140a+, Lymph Node, avg-5</a><br>16: MicroRNA:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/ATGTACA,MIR-493">ATGTACA,MIR-493:MSigDB</a><br>18: Disease:<a href="">Childhood Leiomyosarcoma</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC26minus.html">IC26- </a>(<a href="IC26minus_genes.html">86 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/TAKEDA_TARGETS_OF_NUP98_HOXA9_FUSION_16D_DN.html">Genes down-regulated in CD34+ [GeneID=947] hematopoetic cells by expression of NUP98-HOXA9 fusion [GeneID=4928;3205] off a retroviral vector at 16 days after transduction.</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC27plus.html">IC27+ </a>(<a href="IC27plus_genes.html">269 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0008236">serine-type peptidase activity</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0090596">sensory organ morphogenesis</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0009897">external side of plasma membrane</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR001314">Peptidase S1A</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/590">Receptor Tyrosine Kinases|CD molecules|V-set domain containing|I-set domain containing|Immunoglobulin like domain containing</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/ROSS_AML_WITH_PML_RARA_FUSION.html">Top 100 probe sets for pediatric acute myeloid leukemia (AML) subtype t(15;17): has PML RARA fusion [GeneID=5371;5914].</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 2500 k-means-cluster#2</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-92b*&organism=9606">hsa-miR-92b*:mirSVR nonconserved lowEffect-0.1-0.5</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC27minus.html">IC27- </a>(<a href="IC27minus_genes.html">96 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/16109776">Human Leukemia Verhaak05 568genes</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 500 k-means-cluster#3</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC28plus.html">IC28+ </a>(<a href="IC28plus_genes.html">159 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0042445">hormone metabolic process</a><br>9: Interaction:<a href=" ">DDX19B interactions</a><br>14: Coexpression Atlas:<a href="http://www.gudmap.org/">kidney adult RenalCapsule k-means-cluster#3 top-relative-expression-ranked 200</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-6745"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC28minus.html">IC28- </a>(<a href="IC28minus_genes.html">153 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.gudmap.org/">dev gonad e13.5 M SertoliCell Sox9 top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-6857-5p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC29plus.html">IC29+ </a>(<a href="IC29plus_genes.html">143 genes</a>)</td>
<td width=30%>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS50805">KRAB</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE37532_TREG_VS_TCONV_CD4_TCELL_FROM_VISCERAL_ADIPOSE_TISSUE_UP.html">Genes up-regulated in visceral adipose tissue of aged mice: T reg versus T conv.</a><br>16: MicroRNA:<a href=" ">Functional MTI (Weak)</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC29minus.html">IC29- </a>(<a href="IC29minus_genes.html">127 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005539">glycosaminoglycan binding</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/PTEN_DN.V2_UP.html">Genes up-regulated in HCT116 cells (colon carcinoma) upon knockdown of PTEN [Gene ID=5728] by RNAi.</a><br>14: Coexpression Atlas:<a href="http://www.gudmap.org/">dev lower uro neuro e14.5 PelvicGanglion Sox10 top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-3130-3p&organism=9606">hsa-miR-3130-3p:mirSVR nonconserved highEffect-0.5</a><br>18: Disease:<a href="">Chagas Disease</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC30plus.html">IC30+ </a>(<a href="IC30plus_genes.html">220 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0001228">DNA-binding transcription activator activity, RNA polymerase II-specific</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0045165">cell fate commitment</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$E47_02">V$E47 02</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/20421987">Human Lung Hou10 518genes</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium inducedDefinitiveEndoderm fromBlastocyst-derived-iPSC top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4665-5p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC30minus.html">IC30- </a>(<a href="IC30minus_genes.html">116 genes</a>)</td>
<td width=30%>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/TAKEDA_TARGETS_OF_NUP98_HOXA9_FUSION_8D_UP.html">Genes up-regulated in CD34+ [GeneID=947] hematopoetic cells by expression of NUP98-HOXA9 fusion [GeneID=4928;3205] off a retroviral vector at 8 days after transduction.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-127-3p&organism=9606">hsa-miR-127-3p:mirSVR conserved lowEffect-0.1-0.5</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC31plus.html">IC31+ </a>(<a href="IC31plus_genes.html">182 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005201">extracellular matrix structural constituent</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030198">extracellular matrix organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0005275">abnormal skin tensile strength</a><br>6: Domain:<a href="http://www.ebi.ac.uk/interpro/IEntry?ac=IPR009030">Growth fac rcpt</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>9: Interaction:<a href=" ">DCN interactions</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/490">Collagens</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/BOQUEST_STEM_CELL_UP.html">Genes up-regulated in freshly isolated CD31- [GeneID=5175] (stromal stem cells from adipose tissue) versus the CD31+ (non-stem) counterparts.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777046">Stromal Cells, Fi.Sk, gp38+ CD140a+, Skin, avg-4</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-5682"></a><br>18: Disease:<a href="">Ehlers-Danlos Syndrome</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC31minus.html">IC31- </a>(<a href="IC31minus_genes.html">89 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:bone marrow;+AND+sort_by:tissue+specific+score">bone marrow</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC32plus.html">IC32+ </a>(<a href="IC32plus_genes.html">153 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0001228">DNA-binding transcription activator activity, RNA polymerase II-specific</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0009952">anterior/posterior pattern specification</a><br>6: Domain:<a href="http://pfam.sanger.ac.uk/family/PF00046">Homeobox</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">ratio EmbryoidBody vs StemCell top-relative-expression-ranked 500</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC32minus.html">IC32- </a>(<a href="IC32minus_genes.html">81 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium reference SmallAirwayEpithel top-relative-expression-ranked 500</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC33plus.html">IC33+ </a>(<a href="IC33plus_genes.html">223 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0051607">defense response to virus</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0020912">altered susceptibility to Riboviria infection</a><br>6: Domain:<a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=IG">IG</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1269312">Interferon alpha/beta signaling</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE13485_CTRL_VS_DAY7_YF17D_VACCINE_PBMC_DN.html">Genes down-regulated in comparison of unstimulated peripheral blood mononuclear cells (PBMC) versus PBMC 7 days after stimulation with YF17D vaccine.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM605875">Myeloid Cells, Mo.6C+II-.LN, F480+ Ly6c+ MHCII lo/+, Lymph Node, avg-3</a><br>16: MicroRNA:<a href=" ">Functional MTI</a><br>18: Disease:<a href="">Influenza</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC33minus.html">IC33- </a>(<a href="IC33minus_genes.html">114 genes</a>)</td>
<td width=30%>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/VERHAAK_AML_WITH_NPM1_MUTATED_DN.html">Genes down-regulated in acute myeloid leukemia (AML) patients with mutated NPM1 [GeneID=4869].</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 2500 k-means-cluster#2</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-764-3p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC34plus.html">IC34+ </a>(<a href="IC34plus_genes.html">136 genes</a>)</td>
<td width=30%>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0098858">actin-based cell projection</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE11057_PBMC_VS_MEM_CD4_TCELL_UP.html">Genes up-regulated in comparison of peripheral mononuclear blood cells (PBMC) versus memory T cells.</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:tonsil;+AND+sort_by:tissue+specific+score">tonsil</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-139-3p&organism=9606">hsa-miR-139-3p:mirSVR nonconserved lowEffect-0.1-0.5</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC34minus.html">IC34- </a>(<a href="IC34minus_genes.html">121 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">ratio induced-Ectoderm vs StemCell top-relative-expression-ranked 1000 k-means-cluster#3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4722-5p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC35plus.html">IC35+ </a>(<a href="IC35plus_genes.html">236 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0003810">protein-glutamine gamma-glutamyltransferase activity</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0034330">cell junction organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0045177">apical part of cell</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0000488">abnormal intestinal epithelium morphology</a><br>6: Domain:<a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=Filament">Filament</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1270232">Cell junction organization</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/ONDER_CDH1_TARGETS_2_DN.html">Genes down-regulated in HMLE cells (immortalized nontransformed mammary epithelium) after E-cadhedrin (CDH1) [GeneID=999] knockdown by RNAi.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium reference BronchioEpithel top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.targetscan.org/">hsa-miR-532-3p</a><br>18: Disease:<a href="">Barrett Esophagus</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC35minus.html">IC35- </a>(<a href="IC35minus_genes.html">55 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC36plus.html">IC36+ </a>(<a href="IC36plus_genes.html">186 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0043312">neutrophil degranulation</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030667">secretory granule membrane</a><br>5: Mouse Phenotype:<a href="http://www.informatics.jax.org/javawi2/servlet/WIFetch?page=mpAnnotSummary&id=MP:0001876">decreased inflammatory response</a><br>6: Domain:<a href="http://smart.embl-heidelberg.de/smart/do_annotation.pl?BLAST=DUMMY&DOMAIN=IG">IG</a><br>7: Pathway:<a href="http://www.ncbi.nlm.nih.gov/biosystems/1457780">Neutrophil degranulation</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE22886_NAIVE_BCELL_VS_NEUTROPHIL_DN.html">Genes down-regulated in comparison of naive B cells versus unstimulated neutrophils.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM854338">Myeloid Cells, GN.Arth.SynF, CD11b+ Ly6-G+, Synovial Fluid, avg-3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-6907-5p"></a><br>18: Disease:<a href="">Mucocutaneous Lymph Node Syndrome</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC36minus.html">IC36- </a>(<a href="IC36minus_genes.html">105 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/CHARAFE_BREAST_CANCER_BASAL_VS_MESENCHYMAL_UP.html">Genes up-regulated in basal-like breast cancer cell lines as compared to the mesenchymal-like ones.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP RBC top-relative-expression-ranked 500</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC37plus.html">IC37+ </a>(<a href="IC37plus_genes.html">157 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0038024">cargo receptor activity</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006910">phagocytosis, recognition</a><br>9: Interaction:<a href=" ">KRTAP1-1 interactions</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$PAX_Q6">V$PAX Q6</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM777055">Stromal Cells, FRC.SLN, gp38+ CD31- CD140a+, Lymph Node, avg-4</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-3909"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC37minus.html">IC37- </a>(<a href="IC37minus_genes.html">145 genes</a>)</td>
<td width=30%>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-6134"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC38plus.html">IC38+ </a>(<a href="IC38plus_genes.html">212 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006958">complement activation, classical pathway</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019814">immunoglobulin complex</a><br>10: Cytoband:<a href=" ">14q32.33</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/349">Immunoglobulin heavy locus at 14q32.33</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">Progenitor-Cell-Biology-Consortium inducedDefinitiveEndoderm fromFibroblast-derived-iPSC top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-6965-5p"></a><br>18: Disease:<a href="">Congenital defects</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC38minus.html">IC38- </a>(<a href="IC38minus_genes.html">102 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0006936">muscle contraction</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/19305503">Rat Brain Greco09 425genes</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC39plus.html">IC39+ </a>(<a href="IC39plus_genes.html">162 genes</a>)</td>
<td width=30%>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_ECM_GLYCOPROTEINS.html">Genes encoding structural ECM glycoproteins</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE27092_WT_VS_HDAC7_PHOSPHO_DEFICIENT_CD8_TCELL_UP.html">Genes up-regulated in wildtype cytotoxic T lymphocytes versus those overexpressing phosphorylation deficient form of HDAC7 [GeneID=51564].</a><br>14: Coexpression Atlas:<a href="http://www.facebase.org">FacebaseRNAseq e10.5 Medial Nasal Eminence top-relative-expression-ranked 2500 k-means-cluster#3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4308"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC39minus.html">IC39- </a>(<a href="IC39minus_genes.html">132 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/MEISSNER_BRAIN_HCP_WITH_H3K27ME3.html">Genes with high-CpG-density promoters (HCP) bearing the H3K27 tri-methylation (H3K27me3) mark in brain.</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC40plus.html">IC40+ </a>(<a href="IC40plus_genes.html">152 genes</a>)</td>
<td width=30%>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/NABA_CORE_MATRISOME.html">Ensemble of genes encoding core extracellular matrix including ECM glycoproteins, collagens and proteoglycans</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$AP1_Q4_01">V$AP1 Q4 01</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-202-3p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC40minus.html">IC40- </a>(<a href="IC40minus_genes.html">87 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC41plus.html">IC41+ </a>(<a href="IC41plus_genes.html">137 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0007389">pattern specification process</a><br>14: Coexpression Atlas:<a href="http://www.gudmap.org/">dev gonad e11.5 F GonadVasMes Flk top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-3977"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC41minus.html">IC41- </a>(<a href="IC41minus_genes.html">129 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0004252">serine-type endopeptidase activity</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/HSIAO_LIVER_SPECIFIC_GENES.html">Liver selective genes</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:liver;+AND+sort_by:tissue+specific+score">liver</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4640-5p"></a><br>18: Disease:<a href="">Neutropenia</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC42plus.html">IC42+ </a>(<a href="IC42plus_genes.html">176 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019955">cytokine binding</a><br>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0072676">lymphocyte migration</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0042101">T cell receptor complex</a><br>6: Domain:<a href="http://pfam.sanger.ac.uk/family/PF00041">fn3</a><br>7: Pathway:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/KEGG_CYTOKINE_CYTOKINE_RECEPTOR_INTERACTION.html">Cytokine-cytokine receptor interaction</a><br>10: Cytoband:<a href=" ">14q11</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/371">T-cell receptor alpha locus at 14q11.2</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE26495_NAIVE_VS_PD1HIGH_CD8_TCELL_UP.html">Genes up-regulated in comparison of naive CD8 T cells versus PD-1 high CD8 T cells.</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM476681">gamma delta T cells, Tgd.vg2-.act.Sp, TCRd+ Vg2- CD44+, Spleen, avg-3</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-1258-3p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC42minus.html">IC42- </a>(<a href="IC42minus_genes.html">102 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC43plus.html">IC43+ </a>(<a href="IC43plus_genes.html">182 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/15489886">Human Sarcoma Mason04 549genes</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM791126">Fetal Liver, SC.LTSL.FL, IgM- CD24- CD117+ IL7R- CD150+ CD48- AA4.1+ CD43+, Fetal Liver, avg-3</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-1911&organism=9606">hsa-miR-1911:mirSVR nonconserved highEffect-0.5</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC43minus.html">IC43- </a>(<a href="IC43minus_genes.html">68 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC44plus.html">IC44+ </a>(<a href="IC44plus_genes.html">150 genes</a>)</td>
<td width=30%>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS50026">EGF 3</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE22611_NOD2_TRANSD_VS_CTRL_TRANSD_HEK293_MDP_STIM_2H_DN.html">Genes down-regulated in HEK293 cells at 2h after stimulation by muramyl dipeptide [PubChem=11620162]: over-expressing mutant NOD2 [GeneID=64127] versus control.</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">geo heart top-relative-expression-ranked 100</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-6934-3p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC44minus.html">IC44- </a>(<a href="IC44minus_genes.html">144 genes</a>)</td>
<td width=30%>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE9988_LPS_VS_VEHICLE_TREATED_MONOCYTE_UP.html">Genes up-regulated in comparison of monocytes treated with 1 ng/ml LPS (TLR4 agonist) versus untreated monocytes.</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC45plus.html">IC45+ </a>(<a href="IC45plus_genes.html">155 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0030198">extracellular matrix organization</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0062023">collagen-containing extracellular matrix</a><br>6: Domain:<a href="http://pfam.sanger.ac.uk/family/PF07686">V-set</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/590">Receptor Tyrosine Kinases|CD molecules|V-set domain containing|I-set domain containing|Immunoglobulin like domain containing</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/POOLA_INVASIVE_BREAST_CANCER_UP.html">Genes up-regulated in atypical ductal hyperplastic tissues from patients with (ADHC) breast cancer vs those without the cancer (ADH).</a><br>14: Coexpression Atlas:<a href="http://www.gudmap.org/">DevelopingLowerUrinaryTract adult ureter emap-29479 top-relative-expression-ranked 500</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC45minus.html">IC45- </a>(<a href="IC45minus_genes.html">150 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0099003">vesicle-mediated transport in synapse</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:parathyroid gland;+AND+sort_by:tissue+specific+score">parathyroid gland</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-3106-5p"></a><br></td>
</tr>
<tr>
<td width=10%><a href="IC46plus.html">IC46+ </a>(<a href="IC46plus_genes.html">211 genes</a>)</td>
<td width=30%>6: Domain:<a href="http://www.expasy.org/cgi-bin/nicedoc.pl?PS50835">IG LIKE</a><br>11: Transcription Factor Binding Site:<a href="http://www.broad.mit.edu/gsea/msigdb/cards/V$TAL1BETAE47_01">V$TAL1BETAE47 01</a><br>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/20502458">Human Sarcoma Missiaglia10 176genes</a><br>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:tonsil;+AND+sort_by:tissue+specific+score">tonsil</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=mmu-miR-7239-3p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC46minus.html">IC46- </a>(<a href="IC46minus_genes.html">118 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC47plus.html">IC47+ </a>(<a href="IC47plus_genes.html">210 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0038024">cargo receptor activity</a><br>13: Coexpression:<a href="http://www.ncbi.nlm.nih.gov/pubmed/15785748">Human Uterine Santin05 139genes</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP RBC top-relative-expression-ranked 500</a><br>16: MicroRNA:<a href="http://www.microrna.org/microrna/getTargets.do?matureName=hsa-miR-1914*&organism=9606">hsa-miR-1914*:mirSVR nonconserved highEffect-0.5</a><br>18: Disease:<a href="">Sarcoidosis</a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC47minus.html">IC47- </a>(<a href="IC47minus_genes.html">94 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="https://www.proteinatlas.org/search/tissue_specificity_rna:liver;+AND+sort_by:tissue+specific+score">liver</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4665-5p"></a><br>18: Disease:<a href="">Renal Insufficiency</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC48plus.html">IC48+ </a>(<a href="IC48plus_genes.html">101 genes</a>)</td>
<td width=30%>none</td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC48minus.html">IC48- </a>(<a href="IC48minus_genes.html">84 genes</a>)</td>
<td width=30%>1: GO: Molecular Function:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0005126">cytokine receptor binding</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/GSE15750_DAY6_VS_DAY10_TRAF6KO_EFF_CD8_TCELL_UP.html">Genes up-regulated in comparison of wild type CD8 effector T cells at day 6 versus those  from mice defficient for TRAF6 [GeneID=7189] at day 10.</a><br></td>
</tr>
<tr>
<td width=10%><a href="IC49plus.html">IC49+ </a>(<a href="IC49plus_genes.html">196 genes</a>)</td>
<td width=30%>2: GO: Biological Process:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0008037">cell recognition</a><br>3: GO: Cellular Component:<a href="http://www.ebi.ac.uk/ego/DisplayGoTerm?id=GO:0019814">immunoglobulin complex</a><br>4: Human Phenotype:<a href="null">Increased circulating antibody level</a><br>14: Coexpression Atlas:<a href="http://www.progenitorcells.org/">PP GMP top-relative-expression-ranked 2500 k-means-cluster#2</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-4689"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC49minus.html">IC49- </a>(<a href="IC49minus_genes.html">105 genes</a>)</td>
<td width=30%>none</td>
</tr>
<tr>
<td width=10%><a href="IC50plus.html">IC50+ </a>(<a href="IC50plus_genes.html">166 genes</a>)</td>
<td width=30%>12: Gene Family:<a href="http://www.genenames.org/cgi-bin/genefamilies/set/471">CD molecules|Tumor necrosis factor superfamily</a><br>13: Coexpression:<a href="http://www.broadinstitute.org/gsea/msigdb/cards/POOLA_INVASIVE_BREAST_CANCER_UP.html">Genes up-regulated in atypical ductal hyperplastic tissues from patients with (ADHC) breast cancer vs those without the cancer (ADH).</a><br>14: Coexpression Atlas:<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM605837">Myeloid Cells, DC.8-4-11b-.Sp, CD45-APC CD11b-FITC CD11c-eFluor780 CD4-PE CD8a-eF, Spleen, avg-2</a><br>16: MicroRNA:<a href="http://www.mirdb.org/cgi-bin/mature_mir.cgi?name=hsa-miR-7155-5p"></a><br></td>
<td width=1%>*<br>*</td>
<td width=10%><a href="IC50minus.html">IC50- </a>(<a href="IC50minus_genes.html">95 genes</a>)</td>
<td width=30%>14: Coexpression Atlas:<a href="http://genomebrowser.facebase.org/">e10.5 ColumEpith Maxil top-relative-expression-ranked 500</a><br></td>
</tr>
</table>
</body></html>
